Rakefile

```diff
@@ -72,6 +72,13 @@ namespace :lint do
   task :licenses do
     sh 'bundle exec license_finder'
   end
+
+  begin
+    require 'yard-junk/rake'
+    YardJunk::Rake.define_task
+  rescue LoadError # rubocop:disable Lint/HandleExceptions
+    # Not an issue; this lint tool requires Ruby 2.3+
```

Comment:
I don't like "nulling" rescues of lib loading. This might result in a search for a failure. Please output at least a warning. Which then can be found on CI - if we choose to add it to the test pipeline.

Reply:
I used `Kernel#warn`, is that acceptable?